### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by ice949**
### New Features ###
- Initialise React app
- Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two <Route>s and <Link>s for the app's navigation:

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+const Book = () => (
+  <>
+    <div className="card">
+      <div className="title">
+        <span>Action</span>
+        <h1>Kung Fu Panda: The Dragon Knight</h1>
+        <p>Unseen Content</p>
+        <ul>
```
Comment (Ugboaja-Uchechi) [inline src/components/Book.js]: - [ ] Great job 👏 . Please, create a prop for the book component and it should render the title and author of the book as a prop. 🙏 . You can read more about props in this [article](https://reactjs.org/docs/components-and-props.html) and [video](https://youtu.be/PHaECbrKgs0)

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @ice949, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Creating more commit messages so as to easily keep track of changes in your code 🙏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mwanawabangona — APPROVED** [review]
Hi @ice949  ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ice949** [comment]
Thanks @mwanawabangona

**Pull request merged**